conanfile.py

```diff
@@ -2,29 +2,42 @@
 
 class PahomqttclibConan(ConanFile):
     name = "pahomqttc"
-    license = "<Put the package license here>"
-    author = "<Put your name here> <And your email here>"
-    url = "<Package recipe repository url here, for issues about the package>"
+    version = "1.2.1"
     description = "This recipe file used to build and package binaries of pahomqttc repository"
     topics = ("<Put some tag here>", "<here>", "<and here>")
-    settings = "os", "compiler", "build_type", "arch"
-    options = { "shared": [True, False] }
-    default_options = "shared=False"
+    url = "https://github.com/elear-solutions/paho.mqtt.c"
+    license = "<Put the package license here>"
     generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+    }
+    default_options = {key: False for key in options.keys()}
+    default_options ["shared"] = False
 
-    def build(self):
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def _configure_cmake(self):
         cmake = CMake(self)
         if (self.settings.os == "Android"):
             cmake.definitions["Platform"] = "android"
```

Comment:
remove if and replace the block with cmake.definitions["Platform"] = self.settings.os